src/haskell/install.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -e
+
+GHCUP_VERSION="0.1.19.2"
+GHCUP_BIN="x86_64-linux-ghcup-${GHCUP_VERSION}"
+BOOTSTRAP_HASKELL_GHC_VERSION="${VERSION:-"recommended "}"
+
+# Maybe install curl, gcc, make
+for x in curl gcc make; do
+    which $x > /dev/null || (apt update && apt install $x -y -qq)
+done
+
+GHCUP_DIR=~/.ghcup/bin
```

Comment:
```suggestion
GHCUP_DIR=${_REMOTE_USER_HOME}/.ghcup/bin
```

For the failing MCR tests https://github.com/devcontainers/features/actions/runs/4369291649/jobs/7642925890#step:4:358

I think, ghcup is getting installed at `/root/*` but is the tests are checking at `/home/vscode/*`
The suggestion should help.

Reply:
Done.